From d310f0457ea81eae4973ecc390695fb12b2d1c8f Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Mon, 19 Jun 2017 19:39:36 +0900
Subject: [PATCH 2555/2748] HADOOP-14540. Replace MRv1 specific terms in
 HostsFileReader. Contributed by hu xiaodong.

(cherry picked from commit 300804595e5b8fd447ecb530fcd0ae5b0e59bb63)
(cherry picked from commit 6d82db40d59684fdb7fcdc6159a975b3ca483c27)
(cherry picked from commit 4250250c7bb0837054e7ee0ff9c4ae63bfdd728a)

Change-Id: I63c176dd977f9ea37f245ab056cf8e3bd811acc1
---
 .../org/apache/hadoop/util/HostsFileReader.java    |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/HostsFileReader.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/HostsFileReader.java
index 80de673..4238048 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/HostsFileReader.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/HostsFileReader.java
@@ -28,8 +28,8 @@
 import org.apache.hadoop.classification.InterfaceAudience.Private;
 import org.apache.hadoop.classification.InterfaceStability;
 
-// Keeps track of which datanodes/tasktrackers are allowed to connect to the 
-// namenode/jobtracker.
+// Keeps track of which datanodes/nodemanagers are allowed to connect to the
+// namenode/resourcemanager.
 @InterfaceAudience.LimitedPrivate({"HDFS", "MapReduce"})
 @InterfaceStability.Unstable
 public class HostsFileReader {
-- 
1.7.9.5

